tv_apps/fling-player/test/unit/mock_video_element.js

```diff
@@ -8,25 +8,34 @@
   'use strict';
 
   function MockVideoElement(config = {}) {
+    var self = this;
     this.src = config.src || '';
     this.ended = config.ended || false;
     this.paused = config.paused || true;
     this.hidden = config.hidden || false;
+    this.buffered = [];
     this.duration = config.duration || 0;
     this.currentTime = config.currentTime || 0;
+    this.buffered = {
```

Comment:
What's the purpose of changing this mockfile?
If it's related to other or follow-up test cases, I would suggest to commit them there.